packages/terra-application-navigation/src/ApplicationNavigation.jsx

```diff
@@ -343,7 +343,7 @@ const ApplicationNavigation = ({
 
   useEffect(() => {
     const forceCloseMenu = () => {
-      setDrawerMenuIsOpen(false);
+      updateDrawerIsOpen(false);
```

Comment:
> If we go with option 2, should we retain the previous value of drawerMenuIsOpen and verify that was changed before calling the callback to prevent that possibility?

I think that makes sense and would be the safest option. No one likes debugging infinite loops.

Reply:
Made the change with a slight adjustment - https://github.com/cerner/terra-framework/pull/1451/commits/2f75ccb49365ab76b1b6e3744f0d466ce3270a22
I check and use the `drawerMenuIsOpenRef` current value in the layoutEffect - removing the use of `updateDrawerIsOpen` altogether and calling only `setDrawerIsOpen` in its place

Reply:
Looks like the previous Drawer Closed snapshot still showed the drawer as open since it didn't trigger the [`onDrawerMenuStateChange` that was set](https://github.com/cerner/terra-framework/blob/terra-application-navigation%401.54.0/packages/terra-application-navigation/src/terra-dev-site/test/application-navigation/ApplicationNavigation.test.jsx#L145) - the 'The drawer is open' text should be expected to be missing once the menu is closed.